waspc/src/Wasp/Generator/ExternalConfig/PackageJson.hs

```diff
@@ -0,0 +1,56 @@
+module Wasp.Generator.ExternalConfig.PackageJson
+  ( validatePackageJson,
+  )
+where
+
+import qualified Data.Map as M
+import qualified Wasp.ExternalConfig.PackageJson as P
+import Wasp.Generator.Common (prismaVersion)
+import Wasp.Generator.WebAppGenerator.Common (reactRouterVersion)
+
+validatePackageJson :: P.PackageJson -> IO [String]
```

Comment:
Looking at it, this actually isn't `IO` method, is it? It does only pure calculations. So why do we have it return `IO`? Is it a leftover / mistake?